authentication/index.html

```diff
@@ -0,0 +1,80 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="Looking for Frieda | Log in" content="Looking for Frieda log in page" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.6.0/css/bootstrap.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    />
+    <link rel="stylesheet" href="../dist/css/index.css" />
+    <title>Authentication page</title>
+  </head>  
+  
+  <body class="body-login">
+    <nav class="navbar navbar-expand-lg navbar-custom pt-5 pb-5">
+      <img class="logo" src="/images/logo.png" alt="logo of Looking for Frieda">
+    </nav>
+    
+    <!-- form validation -->
+    <div class="container">
+      <div class="row justify-content-center align-items-center min-vh-100">
+        <div class="col-12col-md-8">
+        </div>
+        <div class="col-12 col-md-8">
+          <!-- Form Container -->
+    <div class="container">
+      <div class="row justify-content-center align-items-center min-vh-100">
+        <div class="col-12 col-md-10 mx-0">
+          <div class="form-container px-5 py-5">
+            <div class="form-wrapper">
+              <h1 class="text-center form-title mb-2">Log in</h1>
+
+              <!-- submission form data will be sent to profile page -->
+              <form action="/profile/index.html" method="post">
+                <div class="form-floating mb-3">
+                  <input
+                    type="email"
+                    class="form-control"
+                    id="email"
+                    name="email"
+                    placeholder="name@example.com"
+                    required
+                  />
+                  <label for="email">Email address</label>
+                </div>
+
+                <div class="form-floating mb-3">
+                  <input
+                    type="password"
+                    class="form-control"
+                    id="password"
+                    name="password"
+                    placeholder="minimum 8 characters"
+                    minlength="8"
+                    required
+                  />
+                  <!-- required min 8 characters -->
+                  <label for="password">Password</label>
+                </div>
+
+                <button type="submit" class="btn btn-block btn-outline-light">
+                  Log in
+                </button>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.slim.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.1/umd/popper.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.6.0/js/bootstrap.min.js"></script>
```

Comment:
If you swap this for bootstrap.bundle, you only need one script tag:

`    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-HwwvtgBNo3bZJJLYd8oVXjrBZt8cqVSpeBNS5n7C8IVInixGAoxmnlMuBnhbgrkm" crossorigin="anonymous"></script>`